src/main/js/controls/Graph/Graph.js

```diff
@@ -393,6 +396,7 @@ class Graph extends Construct {
      * @returns {Graph} - Graph instance
      */
     destroy() {
+        destroyTooltipDiv();
```

Comment:
Can we use the `d3RemoveElement` function for this? instead of a newly created function?

Reply:
I tried doing that, Since we are creating div element outside of container d3RemoveElement will not work for tooltip div element.

Reply:
Can we move step removing the div element below the detachEventHadnlers function call? So we make sure that to make it the consistency with the removing other elements

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/272/commits/6777bd4eee1600694c22412d3e110957caaadfb1#diff-4c7acbc816e711a56dc8fffdd18a4149R400)